spec/components/topic_list_spec.rb

```diff
@@ -63,3 +142,31 @@ def component_name
     expect(rendered).to have_selector(".govuk-\\!-margin-bottom-6")
   end
 end
+
+# RSpec.describe "topic_list component", type: :view do
```

Comment:
This needs removing?

Reply:
Whoops, thanks for spotting, will remove 👍